.travis.yml

```diff
@@ -19,12 +19,7 @@ cache:
 install:
   - npm install
   - lerna bootstrap
-  - cd packages/aeproject-cli
-  - yarn link
-  - cd ..
-  - cd aeproject-lib
-  - npm run tsc
-  - cd ../..
+  - npm run develop
```

Comment:
should we leave the tsc command, because the "npm run develop" script doesn't cover this?